common-library/src/main/java/com/yas/commonlibrary/exception/ApiExceptionHandler.java

```diff
@@ -85,6 +86,27 @@ protected ResponseEntity<ErrorVm> handleInternalServerErrorException(InternalSer
         return ResponseEntity.internalServerError().body(errorVm);
     }
 
+    @ExceptionHandler(MissingServletRequestParameterException.class)
+    protected ResponseEntity<ErrorVm> handleMissingParams(MissingServletRequestParameterException e) {
+        return handleBadRequest(e, null);
+    }
```

Comment:
Hi @khanhduzz,

I noticed that the implementation of this handler method is similar to the handlers for `DuplicatedException` and `DataIntegrityViolationException`. Would it be possible to group them into a single handler method?